frontend/app/src/components/molecules/nav-bar/banner.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Button } from '@/components/ui/button';
+
+export interface BannerProps {
+  message: React.ReactNode;
+  type?: 'info' | 'warning' | 'success' | 'error';
+  actionText?: string;
+  onAction?: () => void;
+  showBanner?: boolean;
+}
+
+export const Banner: React.FC<BannerProps> = ({
+  message,
+  type = 'info',
+  actionText,
+  onAction,
+  showBanner = true,
+}) => {
+  if (!showBanner) {
+    return null;
+  }
```

Comment:
I think this should happen at the level of the parent component - e.g. if we're going to pass `showBanner={someBooleanState}` we should instead do `!someBooleanState && <Banner ...>`

Reply:
agreed, was playing with making the banner hideable but removed this